stock_getter/backend_stockgetter/manage.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python
+"""Django's command-line utility for administrative tasks."""
+import os
+import sys
+
+
+def main():
+    """Run administrative tasks."""
```

Comment:
Curious why triple quotes here?

Reply:
triple quote is a multiline comment in Python

Reply:
isn't this single line comment

Reply:
it is for now :)

Reply:
also, just occurred to me that triple quotes are used to define a function docstring, so I'm inclined to believe that's why it's here

Reply:
ok